apps/statesvisited/states_visited.star

```diff
@@ -0,0 +1,135 @@
+    "WV": "iVBORw0KGgoAAAANSUhEUgAAAHgAAABACAYAAADRTbMSAAAABmJLR0QA/wD/AP+gvaeTAAABT0lEQVR4nO3VsUsCYRyH8Tu9ri65A02yQKKoMYkaHJoqEqLmpsgxCvoXnJvaGqLJqSmICMSGxqCo1jKkMAgCQ83z1A4zWgoaKu4geDt4PtvvnR74Dq8kAQAAAAAAAAAAAAAAAAAAAK4tjYQM0Q1u+EQHeM1kqHdBdIMbiugArxkMqHOZmdFwqd7OqYocHNC1+XzN2kqe3FyIbvsOA7twMB1LxXv0hE+WEhW9ZRqqoh8/llKnz+Wc6LafMLBDmanYejxsrMkfd1DtMO4t+/zowd5NF55ehMb9gj/YgcVoVBs2tFX5y1u+2jzcLpaT6ULh344rSQzsiCmZWqfPr3/etVa7kS02NnYu76oiu5zwiw7wglvTbs72h7r7utQJ6/WtflWp7a+cXe+J7sIf2xwfGluORAKiOwAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAHvQOBAVNj5dB4UAAAAAASUVORK5CYII=",
+    "WY": "iVBORw0KGgoAAAANSUhEUgAAAHgAAABACAYAAADRTbMSAAAABmJLR0QA/wD/AP+gvaeTAAABfklEQVR4nO3Yy0ojQRjF8VO3ZIgBxWvC0CjuIq4Fl77KvIZv4ps4zEuIOzeOOANegnYEIZ1L1axmdmJDJhVI/r9VLz74DhwKqloCAAAAAAAAAACYgVvATr9+dHrWbDZ89fY6kJQWkGFlmNwL20XveO3w5Lskpcm4UopPKaVfUryPw+HDtHp/TKPB72F597Pq929y51s2PvfCsN7Z//ttfGhKKoxUSDq1jZa8NiUVanZ7afL2fPlydfktd8ZlYrNv9K39z4ckY52xoVHOO86yy16w865WwZKUonmYZ5ZVkP8EW3dQd3SaxhQ8o/wFm/on2MaKgmeUt+CNgw0XGt2647H/+jjPOKsga8Gb27s9uRDqzKbpZFq+397NO9Oyy/xMGt5X5fO5c37H2NSRdXvJ+j0r7ZrwZctY9+9dHsejvsqSW/SMsv/o+Ei73dnx28XX2Gp1bVjrpDgKg+sfF4vOBQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAOD/+wNa+V2DrigExgAAAABJRU5ErkJggg==",
+}
+
+STATE_NAMES = {
+    "AL": "Alabama",
+    "AK": "Alaska",
+    "AZ": "Arizona",
+    "AR": "Arkansas",
+    "CA": "California",
+    "CO": "Colorado",
+    "CT": "Connecticut",
+    "DE": "Delaware",
+    "FL": "Florida",
+    "GA": "Georgia",
+    "HI": "Hawaii",
+    "ID": "Idaho",
+    "IL": "Illinois",
+    "IN": "Indiana",
+    "IA": "Iowa",
+    "KS": "Kansas",
+    "KY": "Kentucky",
+    "LA": "Louisiana",
+    "ME": "Maine",
+    "MD": "Maryland",
+    "MA": "Massachusetts",
+    "MI": "Michigan",
+    "MN": "Minnesota",
+    "MS": "Mississippi",
+    "MO": "Missouri",
+    "MT": "Montana",
+    "NE": "Nebraska",
+    "NV": "Nevada",
+    "NH": "New Hampshire",
+    "NJ": "New Jersey",
+    "NM": "New Mexico",
+    "NY": "New York",
+    "NC": "North Carolina",
+    "ND": "North Dakota",
+    "OH": "Ohio",
+    "OK": "Oklahoma",
+    "OR": "Oregon",
+    "PA": "Pennsylvania",
+    "RI": "Rhode Island",
+    "SC": "South Carolina",
+    "SD": "South Dakota",
+    "TN": "Tennessee",
+    "TX": "Texas",
+    "UT": "Utah",
+    "VT": "Vermont",
+    "VA": "Virginia",
+    "WA": "Washington",
+    "WV": "West Virginia",
+    "WI": "Wisconsin",
+    "WY": "Wyoming",
+}
+
+def main(config):
+    return render.Root(
+        render.Stack(
+            [
+                render.Image(base64.decode(STATES[state]), 64)
+                for state in STATES
+                if config.get(state) and config.get(state) != "false"
```

Comment:
There is a small bug in the `pixlet serve` tool. If you are on a URL like `http://localhost:8080/?FL=false`, the string `"false"` gets into the `config` object on this line. This is bad because the string `"false"` is truthy in Pythonic languages (all strings are True except the empty string).

I will file a bug to track this issue.

Reply:
This behaviour is actually intentional. Check out `config.bool()` to handle type conversion for you. You can also do something like `config.get("FL") == "true"`.